ledger/participant-state/kvutils/tools/engine-benchmark/src/benchmark/scala/ledger/participant/state/kvutils/tools/engine/benchmark/Replay.scala

```diff
@@ -83,6 +83,7 @@ class Replay {
   def bench(): Unit = {
     val result = engine
       .replay(
+        benchmark.transaction.tx.submitter.toOption.get,
```

Comment:
Is this `Option` ever `None`? Would it make sense to provide a bit more context by throwing an exception in case it is instead of relying on the assertion of `Option.get` failing?

Reply:
Fixed in a proper better way.